omdb/models.py

```diff
@@ -80,6 +80,7 @@ class Item(Storage):
         'Runtime',
         'Season',
         'seriesID',
+        'totalSeasons',
```

Comment:
My suggestion was to add both `totalSeasons` and `totalResults` but because I don't have an apikey to use, I can't verify what the data returned looks like. Although, if `totalResults` is being returned at the same level as `Search`, then some other changes would be needed since omdb.py pulls the `Search` field from the response and just returns the list of results. So perhaps hold off on `totalResults` for now.